src/libs/ttffont.js

```diff
@@ -898,9 +928,10 @@ var CmapTable = (function(_super) {
       i = 0 <= tableCount ? ++i : --i
     ) {
       entry = new CmapEntry(data, this.offset);
+      if (Object.keys(entry.codeMap).length === 0) continue;
```

Comment:
When TTF has the unsupported version of format CMAP, the `entry` with an empty `codeMap` is created, so I skip it.